### DISCUSSION TITLE
How to log in kyuubi dashboard ui?

### DISCUSSION THREAD

**Discussion by eubnara** [apache/kyuubi#6231] [Q&A]
```
kyuubi.authentication                    LDAP,KERBEROS
kyuubi.frontend.rest.bind.port           10099
```

I run kyuubi server with above configurations.
When I connect to kyuubi server with 10099 port, nothing has been shown.
![image](https://github.com/apache/kyuubi/assets/12639125/43e1323e-1eee-4eb9-8411-4981a037cfab)
![image](https://github.com/apache/kyuubi/assets/12639125/9757398e-7ff1-467d-bb8d-df8694c5b965)

When I see developer tools, I find 401 responses.
However, on UI there is no login options.
I got kerberos tgt but it seems that spnego mechanism doesn't work.

**Comment by pan3793** [comment]
It has not been implemented yet, for basic authN, is tracked in https://github.com/apache/kyuubi/issues/6079
